scripts/version

```diff
@@ -8,17 +8,19 @@ if [ $# -ne 0 ]; then
   echo 'warning: this script does not take any argument.' >&2
 fi
 
+DATE_FMT="%Y-%m-%d"
+SOURCE_DATE_EPOCH="${SOURCE_DATE_EPOCH:-$(date +%s)}"
+date=$(date -u -d "@$SOURCE_DATE_EPOCH" "+$DATE_FMT" 2>/dev/null || date -u -r "$SOURCE_DATE_EPOCH" "+$DATE_FMT" 2>/dev/null || date -u "+$DATE_FMT")
```

Comment:
The `-r` is for [BSD date](https://www.freebsd.org/cgi/man.cgi?date)
and the last part is just as a fallback in case `date` does not understand -d or -r options.

If it is OK to just work with GNU date, we can use a much simpler version of the command.